src/v/cluster/node/local_monitor.cc

```diff
@@ -89,6 +92,30 @@ ss::future<struct statvfs> local_monitor::get_statvfs(const ss::sstring& path) {
     }
 }
 
+float percent_free(const disk& disk) {
```

Comment:
it looks like this should have file scoping (ie static or in an anonymous namespace?)